plugins/s3-viewer-backend/package.json

```diff
@@ -31,17 +31,17 @@
   },
   "dependencies": {
     "@aws-crypto/sha256-browser": "^2.0.1",
-    "@aws-sdk/protocol-http": "^3.127.0",
-    "@aws-sdk/signature-v4": "^3.130.0",
-    "@backstage/backend-common": "^0.18.2",
-    "@backstage/backend-plugin-api": "^0.4.0",
-    "@backstage/backend-tasks": "^0.4.3",
-    "@backstage/config": "^1.0.6",
-    "@backstage/errors": "^1.1.4",
-    "@backstage/plugin-auth-node": "^0.2.11",
-    "@backstage/plugin-permission-backend": "^0.5.17",
-    "@backstage/plugin-permission-common": "^0.7.3",
-    "@backstage/plugin-permission-node": "^0.7.5",
+    "@aws-sdk/protocol-http": "^3.282.0",
+    "@aws-sdk/signature-v4": "^3.282.0",
```

Comment:
Similar to the ceph update above, this should be in its own commit in the future.

Reply:
I've just upgraded the commits. Now they're separated from backstage-internals and other dependencies